apps/agent/src/main/init/ipcMain.ts

```
@@ -14,12 +14,13 @@ import {
 	UserService,
 	pluginListeners
 } from '@gauzy/desktop-lib';
-import { getApiBaseUrl, delaySync } from '../util';
+import { getApiBaseUrl, delaySync, getAuthConfig } from '../util';
 import { startServer } from './app';
 import AppWindow from '../window-manager';
 import * as moment from 'moment';
 import * as path from 'path';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)